journal/original/02-chapter-1.adoc

```diff
@@ -80,14 +80,15 @@ A young man, a merchant, then present, who frequented my company,
 and would often speak of the people called Quakers,
 and say their principles were the truth,
 hearing the priest boast of his abusive behaviour to the said two women,
-answered and said.
-It was a poor victory he had gotten over two poor women;
+answered and said,
+"`It was a poor victory he had gotten over two poor women;`"
```

Comment:
Here in the original, I would lean towards just fixing the comma (like you did), but not adding the quotes, and leaving the text the way it is.👍🏻